container-host-files/etc/hcf/config/role-manifest.yml

```diff
@@ -1765,6 +1829,7 @@ configuration:
       value_type: certificate
     description: PEM-encoded CA certificate used to sign TLS certificate hosted by Diego BBS.
     required: true
+  - name: JWT_SIGNING_PUB
```

Comment:
What else other than the description are you looking for?
This is a value auto-retrieved by a script